app/src/layouts/DashboardLayout/DashboardContent.js

```diff
@@ -83,6 +85,17 @@ const DashboardContent = () => {
     }
   }, [location, prevProps, searchParams]);
 
+  useEffect(() => {
+    if (window.location.href.includes(PATHS.SESSIONS.RELATIVE) && refParam === "producthunt") {
+      dispatch(
+        actions.toggleActiveModal({
+          modalName: "authModal",
+          newValue: true,
+        })
+      );
+    }
+  }, [dispatch, refParam]);
+
```

Comment:
Lets add this check only in Session Recording component, instead of adding at top level